mkdocs/docs/HPC/teaching_training.md

```diff
@@ -0,0 +1,108 @@
+
+[^TODO]: insert some links to the interactvie cluster?
+
+In order to help you out, you need to provide following pieces of information
+when you [contact the {{ hpcteam }}]({{ hpc_support_url }}):
+
+* Group moderators and optional UGent course id
+* Start and end date
+* Optional
+  * Share
+  * Reservation
+  * Software
+  * Portal app
+
+## Group moderators
+
+To organise teachers and students, groups are used. Typically the teachers map
+to the role of moderator(s), and students are regular members.
+
+The moderator can invite
+students as member via their vsc account (the students have to accept the invite);
+or the students can invite themself to the group (and moderator has to approve the invites).
+
+You provide us with the vsc account(s) for the moderator(s),
+the moderators can manage the members and moderators themself afterwards.
+
+[^TODO]: do we support initial list of participants?
+
+[^TODO]: we control the groupname prefix? or have them all use the same one (unprotected). incl some convntion like academci year
+
+
+### UGent course id
+
+[^TODO]: check terminology
+
+When teaching a registered UGent course, we can offload the management of the vsc accounts from the students and
+the teachers. We do this by creating a vsc account for each registered student that is linked to the course id
+(if they do not have an account already). And we then populate the group with the vsc accounts of the students
+and the list of moderators. Using this method, you cannot manage the members or moderators yourself:
+group members unknown to the course will be considered unregistered students and will be removed from the group.
+
+The created accounts will be accounts without an ssh-key. This allows the students to use e.g. the portal, but if
+they require ssh access to the infrastructure, they will have to add one themself.
+
+[^TODO]: what does the sync do? can moderators add moderators themself?
+
+## Start and end date
+
+We require an end date to perform any required cleanup of groups and optional data, apps or reservations.
+
+[^TODO]: can we do some automated cleanup for ugent courses? when the course disappears from AD?
+
+[^TODO]: we need a maximum enddate? max one year after the request?
+
+The start date will be `best-effort`, based on the load of the support team and the requirements you have.
+The sooner you make the request, the better.
+
+[^TODO]: do we mention some actual ETA here? like one month upfront?
+
+[^TODO]: when do course of the next academic year appear?
+
+## Share
+
+When required, we can create a directory that is writeable by the moderators and accessible by all
+members and moderators. This directory can be used to share e.g. input files with the students.
+
+The data policy for this share is the following: on the enddate, the directory will be made unaccessible
+nby anyone, and one year after the enddate it will be removed. When the moderators require anything from
```

Comment:
`nby` -> `to`

`enddate` -> `end date`

`it will be removed` -> `the directory will be removed`

(will there be a notification of automatic removal?)